src/Sulu/Bundle/WebsiteBundle/Twig/Content/ContentTwigExtension.php

```diff
@@ -108,6 +118,51 @@ public function loadParent($uuid)
             throw new ParentNotFoundException($uuid);
         }
 
-        return $this->load($node->getParent()->getIdentifier());
+        return $this->load($node->getParent()->getIdentifier(), $properties);
+    }
+
+    private function resolveProperties(StructureInterface $contentStructure, array $properties): array
+    {
+        $contentProperties = [];
+        $extensionProperties = [];
+
+        foreach ($properties as $sourceProperty => $targetProperty) {
+            if (!\is_string($sourceProperty)) {
+                $sourceProperty = $targetProperty;
+            }
+
+            if (!\strpos($sourceProperty, '.')) {
+                $contentProperties[$sourceProperty] = $targetProperty;
+            } else {
+                $extensionProperties[$sourceProperty] = $targetProperty;
+            }
+        }
+
+        $resolvedStructure = $this->structureResolver->resolve(
+            $contentStructure,
+            !empty($extensionProperties),
+            \array_keys($contentProperties)
+        );
+
+        foreach ($contentProperties as $sourceProperty => $targetProperty) {
+            if ($sourceProperty !== $targetProperty) {
```

Comment:
Not all templates will have all source properties, it should not fail when sourceProperties do not exists:

```suggestion
            if (!\array_key_exists($sourceProperty, $resolvedStructure['content'])) {
                $resolvedStructure['content'][$targetProperty] = null;
                $resolvedStructure['view'][$targetProperty] = [];
            } elseif ($sourceProperty !== $targetProperty) {
```

Not sure how this is currently handled in the PageSelection if it is set to null or the property is never set but I think it should be consistent.

Reply:
the `page_selection` just omits the property. implemented it like this here too 🙂